crates/ox_content_ssg/src/ssg.css

```
@@ -627,12 +627,65 @@ a:hover {
+.nav-summary::-webkit-details-marker {
+  display: none;
+}
+.nav-title--summary::before,
+.nav-details > .nav-summary::before {
+  content: "";
+  flex: 0 0 auto;
+  width: 0.4rem;
+  height: 0.4rem;
+  margin-block-start: 0.32rem;
+  margin-inline-end: 0.45rem;
+  border-right: 1.5px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)